bin/deploy

```diff
@@ -157,6 +157,18 @@ fi
 echo "*******************************************************************"
 echo
 
+if [[ $application_name == 'fb-publisher' ]]; then
+  # The clusters (K8S and EKS) have different names for their bearer tokens
+  # So these tokens have been set in the deployment pipeline
+  # Each token is different also per environment (test and live) so
+  # it needs this dynamic grep for K8S_BEARER_TOKEN_TEST and
+  # K8S_BEARER_TOKEN_LIVE
+  bearer_token_env_var_name="K8S_BEARER_TOKEN_${platform_environment}"
+  bearer_token_env_var_name=$(echo $bearer_token_env_var_name | tr '-' '_' | tr [a-z] [A-Z])
```

Comment:
is the `tr '-' '_'` just in case we accidentally add an envar with a `-`?

Reply:
I think I copied from other scripts. Should I removed it?

Reply:
yeah i think we can remove it. publisher (and editor) will only ever be test or live so there won't be a need to swap the `-` 🤞